dbt/include/bigquery/macros/materializations/table.sql

```diff
@@ -63,6 +63,7 @@ df = model(dbt, spark)
 # COMMAND ----------
 # this is materialization code dbt generated, please do not modify
 
+import pyspark
```

Comment:
This is added to avoid error when trying to use pyspark for certain versions of image